bolt/lib/Target/AArch64/AArch64MCPlusBuilder.cpp

```diff
@@ -752,6 +764,19 @@ class AArch64MCPlusBuilder : public MCPlusBuilder {
       return true;
     }
 
+    if (DefJTBaseAdd->getOpcode() == AArch64::ADR) {
+      // TODO: handle that case when we do not have adrp/add pair.
+      // It also occurs when the binary is static.
+      //  adr     x13, 0x215a18 <_nl_value_type_LC_COLLATE+0x50>
```

Comment:
nit: '..handle the pattern where there is no adrp...'

Reply:
Thanks for the review. Done.